cross-chain/arbitrum/README.adoc

```diff
@@ -0,0 +1,42 @@
+:toc: macro
+
+= Threshold cross-chain - Arbitrum
+
+This package brings Bitcoin to Ethereum L2 Arbitrum. For more details please
+see link:https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-8.adoc[Cross-chain Tokenized Threshold BTC]
+
+== How it works?
+
+```
++----------------------------+         +---------------------------------------------------------------------------+
+|          Ethereum          |         |                                Arbitrum                                   |
+|                            |         |                                                                           |
+|  +----------------------+  |         |  +----------------------+  +-------------------------+  +--------------+  |
+|  | Wormhole TokenBridge |--|---------|--| Wormhole TokenBridge |--| ArbitrumWormholeGateway |--| ArbitrumTBTC |  |
+|  +----------------------+  |         |  +----------------------+  +-------------------------+  +--------------+  |
+|                            |         |                                                                           |
++----------------------------+         +---------------------------------------------------------------------------+
+```
+
+- `ArbitrumTBTC` canonical tBTC token on Arbitrum with a minting authority
+delegated to `ArbitrumWormholeGateway`.
+- `ArbitrumWormholeGateway` is a smart contract wrapping and unwrapping 
+Wormhole-specific tBTC representation into the canonical `ArbitrumTBTC` token.
+
+=== Deploy contracts
+
+To deploy all contracts on the given network, please run:
+```
+yarn deploy --network <network>
```

Comment:
Would it be worth listing possible networks (`arbitrumGoerli` and `arbitrumMainnet`)?
And also maybe mention that for local deployment `yarn deploy --network hardhat` should be executed.

Reply:
part of [d4c157b](https://github.com/keep-network/tbtc-v2/pull/554/commits/d4c157b18cc691fbd0b019a2495e6a66ea7557b2)